ui/app/AppLayouts/Profile/views/MyProfileView.qml

```diff
@@ -301,9 +301,9 @@ SettingsContentBase {
 
                 displayName.focus: !isEnsName
                 displayName.input.edit.readOnly: isEnsName
-                displayName.text: profileStore.name
+                displayName.text: profileStore.displayName
                 displayName.validationMode: StatusInput.ValidationMode.Always
-                displayName.validators: isEnsName ? [] : displayNameValidators.validators
+                displayName.validators: isEnsName || (profileStore.displayName === "" && displayName.text === "") ? [] : displayNameValidators.validators
```

Comment:
Probably could be generalized to prevent validating what's already set:
```suggestion
                displayName.validators: isEnsName || (profileStore.displayName === displayName.text) ? [] : displayNameValidators.validators
```